src/main/java/org/folio/rest/impl/OrganizationApi.java

```diff
@@ -33,14 +33,14 @@ public OrganizationApi() {
   }
 
   @Override
-  public void getOrganizationsOrganizations(String query, int offset, int limit, String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    organizationService.getOrganizationCollection(offset, limit, query, lang, vertxContext, okapiHeaders)
+  public void getOrganizationsOrganizations(String query, String totalRecords, int offset, int limit, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```

Comment:
Is totalRecords something new stuff which bringed after raml-util upgrade?

Reply:
Yes

Reply:
please refer to https://issues.folio.org/browse/RMB-724